nestkernel/connection_manager.h

```diff
@@ -175,9 +175,9 @@ class ConnectionManager
 private:
   std::vector< ConnectorModel* > pristine_prototypes_; //!< The list of clean synapse prototypes
   std::vector< std::vector< ConnectorModel* > > prototypes_; //!< The list of available synapse
-                                                             //prototypes: first dimenasion one
-                                                             //entry per thread, second dimantion
-                                                             //for each synapse type
+  // prototypes: first dimenasion one
+  // entry per thread, second dimantion
+  // for each synapse type
```

Comment:
The indentation got skewed. If you indent it again, clang-format will keep the indentation.